command/ca/provisionerbeta/add.go

```diff
@@ -107,7 +113,10 @@ func addCommand() cli.Command {
     : Uses Kubernetes Service Account tokens.
 
     **SSHPOP**
-    : Uses an SSH Certificate / private key pair to sign provisioning tokens.`},
+    : Uses an SSH Certificate / private key pair to sign provisioning tokens.
+
+	**NEBULA**
+	: Uses a Nebula Certificate / private key pair to sign provisioning tokens.`},
```

Comment:
I think it's sensible to write `Certificate` without the capital. Can be changed for `SSHPOP` then too.

Reply:
Sure I didn't notice we were using here different letter cases.

Reply:
Fixed with bf0a16f